hibernate6-jpa/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id 'io.micronaut.build.internal.sql-module'
+}
+
+// TODO: Temporary disabled tests and binary compatibility check
+test.onlyIf { false }
+micronautBuild {
+    binaryCompatibility {
+        enabled = false
+    }
+}
+
+dependencies {
+    annotationProcessor(mn.micronaut.graal)
+
+    // included for Java 11
+    runtimeOnly(libs.jaxb.api)
+
+    api(libs.managed.hibernate6.core) {
+        exclude group:'org.jboss.spec.javax.transaction', module:'jboss-transaction-api_1.2_spec'
+        exclude group:'org.javassist', module:'javassist'
+        exclude group:'org.jboss:jandex', module:'jar'
+    }
+    api(libs.jakarta.transaction.api)
+    api(mn.micronaut.validation)
+    // TODO: This would pull in hibernate5 so we want to exclude it for now
+    // Later will include mnData.micronaut.data.tx.hibernate6 after micronaut-data supports Hibernate6
+    api(mnData.micronaut.data.tx.hibernate) {
```

Comment:
Remove this dependency for now

Reply:
It was needed to compile, but I have added one temp class just to remove this. Later that temp class will be replaced, based on TODO comment.